### PR TITLE
rc V0.0.3

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Addresses #35 and failure to display description on pypi.  Also fixes errors in zenodo.json.

# Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Apparently we can't test these things without making a release.  Maybe need to find a code checker for .json and .cfg files.

# Checklist:

- [x] Update the version number
- [x] Verify and update Zenodo information in .zeondo.json: https://jsonlint.com/
- [x] doc tests pass locally
- [x] activate this branch on readthedocs (triggered)
- [x] Upload the test distribution to the testPyPi server (https://test.pypi.org/project/pysatSpaceWeather/0.0.3/)
- [x] Contacted developers and users about the future release
- [x] Performed integration tests with downstream packages (include list of these packages)
- [x] Create a wiki page with the release text (https://github.com/pysat/pysatSpaceWeather/wiki/v0.0.3)
- [x] After approval, confirm all changes from the RC pull are listed in the wiki release text.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

**Comment by aburrell** [comment]
Ok, change in requirements and setup.cfg initiated after travis error and feedback from this thread: https://github.com/pypa/setuptools/issues/1951.

**Comment by aburrell** [comment]
Note the changes to the release checklist (also made on pysat wiki).  Hopefully this will help in future :P.

**Comment by jklenzing** [comment]
https://test.pypi.org/project/pysatSpaceWeather/0.0.3/

File: setup.cfg [hunk 1]
```diff
@@ -2,41 +2,46 @@
...
-  Operating System :: POSIX :: Linux
+    Development Status :: 3 - Alpha
+    Topic :: Scientific/Engineering :: Physics
+    Topic :: Scientific/Engineering :: Atmospheric Science
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Natural Language :: English
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
```
Comment (jklenzing) [inline setup.cfg:23]:
```suggestion
    Programming Language :: Python :: 3.6
    Programming Language :: Python :: 3.7
    Programming Language :: Python :: 3.8
```
Should match supported tests on travis

**Review by jklenzing — APPROVED** [review]
minor update to metadata in config

**Review by rstoneback — APPROVED** [review]

**Pull request merged**